cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -655,13 +662,38 @@ public void requestLocationPermission() {
         ActivityCompat.requestPermissions(getCurrentActivity(), new String[]{Manifest.permission.ACCESS_FINE_LOCATION}, locationPermissionRequestCode);
     }
 
+    /**
+     * request for push notification
+     */
+    public void requestPushNotificationPermission() {
+        if (Build.VERSION.SDK_INT >= Constants.ANDROID_13_VERSION) {
```

Comment:
if we dont anything here to be done if its Version < 13
then we should early exit Build.VERSION.SDK_INT < Constants.ANDROID_13_VERSION
otherwise we can combine the first 2 if's to Build.VERSION.SDK_INT >= Constants.ANDROID_13_VERSION && this.hasNotificationPermission()

Reply:
combine:

Combine the 2 if's please like kevin wrote

Reply:
Okay It's done